Raven.Database/Raft/ClusterManagementHttpClient.cs

```diff
@@ -345,7 +345,20 @@ public Task SendNodeUpdateAsync(NodeConnectionInfo node)
                 }
                 else
                 {
+                    // before we send remove from cluster wait until configuration will be propaged
```

Comment:
Okay, but can we avoid doing a hard wait?
This is going to take a minimum of 4x heartbeat time. And it doesn't check  that things actually work.
We have a way to wait for topology changes already, no?